kythe/go/extractors/config/smoke/smoke_test.go

```diff
@@ -159,7 +160,7 @@ func verifyRepoCoverage(t *testing.T, te testExtractor, expectedDownloadCount, e
 		repoFetcher: te,
 	}
 
-	r, err := h.TestRepo("foo")
+	r, err := h.TestRepo(context.TODO(), "foo")
```

Comment:
Can you not pass along the caller's `ctx` here? (If not, please add a comment to say why)

Reply:
Sorry what caller's `ctx`? In the unit test I'm not sure what the deal is.

Reply:
Sorry, I didn't expand the clicky, and misread the code.

Reply:
Ok to resolve this using `s/TODO/Background/`